dogfood/main.tf

```diff
@@ -113,6 +69,47 @@ data "coder_git_auth" "github" {
 
 data "coder_workspace" "me" {}
 
+module "dotfiles" {
+  source   = "https://registry.coder.com/modules/dotfiles"
+  agent_id = coder_agent.dev.id
+}
+
+module "git-clone" {
+  source   = "https://registry.coder.com/modules/git-clone"
+  agent_id = coder_agent.dev.id
+  url      = "https://github.com/coder/coder"
+  path     = data.coder_parameter.repo_dir.value
+}
+
+module "personalize" {
+  source   = "https://registry.coder.com/modules/personalize"
+  agent_id = coder_agent.dev.id
+}
+
+module "code-server" {
+  source   = "https://registry.coder.com/modules/code-server"
+  agent_id = coder_agent.dev.id
+}
+
+module "jetbrains_gateway" {
+  source            = "https://registry.coder.com/modules/jetbrains-gateway"
+  agent_id          = coder_agent.dev.id
+  agent_name        = "dev"
+  project_directory = replace(data.coder_parameter.repo_dir.value, "/^~\\//", "/home/coder/")
+  jetbrains_ides    = ["GO", "WS"]
+  default           = "GO"
+}
+
+module "vscode" {
+  source   = "https://registry.coder.com/modules/vscode-desktop"
+  agent_id = coder_agent.dev.id
+}
+
+module "filebrowser" {
+  source   = "https://registry.coder.com/modules/filebrowser"
+  agent_id = coder_agent.dev.id
```

Comment:
Should we set this to `coder_agent.dev.dir`? Same for code-server, etc. to match old behavior.

Reply:
I agree. I will update.

Reply:
I'm fine with keeping filebrowser in `$HOME`, but `code-server` used to open in the repo dir (and maybe vscode desktop too?).

Reply:
`vscode-desktop` module currently does not support setting a folder so that it will prompt for a folder on launch. I wonder why @kylecarbs did this way.

Reply:
`code-server` does have `folder` variable that we can set to the repo_dir. I will update